services/ui-src/src/forms/mcpar/mcpar.json

```diff
@@ -2983,12 +2983,36 @@
                 }
               },
               {
-                "id": "sanction_remediationDate",
-                "type": "date",
-                "validation": "date",
+                "id": "sanction_remediationCorrected",
```

Comment:
Nit: I would maybe name this one `sanction_remediationCompleted`. Because remediation is the act of correcting something, "remediationCorrected" reads like "correctionCorrected", which is kinda weird imo. I also think `Remediation date non-compliance was corrected` doesn't make any sense, but that one's out of our hands. 🤷